### PR TITLE
Removes `KeyMap.info` from sheets

### PR DESCRIPTION
**Pull request by frzyc**
## Describe your changes

Depends on https://github.com/frzyc/genshin-optimizer/pull/1882,
Removes `KeyMap.info` from sheets.

## Issue or discord link

- 

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

**Comment by nguyentvan7** [comment]
conflicts

**Comment by frzyc** [comment]
> conflicts

Yeah, its weird, Its literally a stacked commit with no changes in between...

File: apps/frontend/src/app/Data/Characters/Alhaitham/index.tsx [hunk 1]
```diff
@@ -158,7 +157,7 @@ const c4MirrorsConsumed_eleMasDisp = infoMut(
       naught
     )
   ),
-  { ...KeyMap.info('eleMas'), isTeamBuff: true }
+  { ...{ path: 'eleMas' }, isTeamBuff: true }
```
Comment (nguyentvan7) [inline apps/frontend/src/app/Data/Characters/Alhaitham/index.tsx:160]: funky find-and-replace I assume?

**Review by nguyentvan7 — APPROVED** [review]
There are a few of these funky find-and-replace bits. but probably fine

**Pull request merged**